src/styles/blocks/header.scss

```diff
@@ -0,0 +1,39 @@
+.header {
```

Comment:
if header and nav are 2 different BEM-blocks, you need to create a new file for each of them